core/themes/basis/template.php

```diff
@@ -38,6 +38,21 @@ function basis_preprocess_page(&$variables) {
     $variables['classes'][] = 'view-name-' . $view->name;
   }
 
+  // Get the installed version.
+  $update_version = config_get('basis.settings', 'css_update_version');
+  $update_preference = config_get('basis.settings', 'css_update');
+
+  // Process supplemental CSS versions.
+  $update_css_versions = basis_css_versions_updated();
+  foreach ($update_css_versions as $update_css_version) {
+    if ($update_css_version !== 'No updates') {
```

Comment:
Save as an empty string so this just becomes:

```suggestion
    if ($update_css_version) {
```